client/reader/embed-helper.jsx

```diff
@@ -87,6 +92,10 @@ function resolveEmbedConfig( embed ) {
 	return embedsConfig.default;
 }
 
+function isPercentage( val ) {
+	return 'string' === typeof val && '%' === val[ val.length - 1 ];
```

Comment:
Good call. Found [an npm module](https://www.npmjs.com/package/percentage-regex) that add verification that the values are digits: 09b6a5c95a2eaa34b52201da4f798e0e979b4358